sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -3030,6 +3057,29 @@ object Radar {
         }
     }
 
+    internal fun warmupStandardIntegrityTokenProvider(config: RadarConfig) {
+        if (config.googlePlayProjectNumber == null) {
+            logger.e("warmupStandardIntegrityTokenProvider: googlePlayProjectNumber is null!");
+            return;
+        } else {
+            // Create an instance of a standard integrity manager
+            val standardIntegrityManager = IntegrityManagerFactory.createStandard(context)
+            standardIntegrityManager.prepareIntegrityToken(
+                StandardIntegrityManager.PrepareIntegrityTokenRequest.builder()
+                    .setCloudProjectNumber(config.googlePlayProjectNumber)
+                    .build()
+            )
+                .addOnSuccessListener { tokenProvider ->
+                    Radar.standardIntegrityTokenProvider = tokenProvider
+                    logger.d("successful warm up of the integrity token provider")
+                }
+                .addOnFailureListener { exception ->
+                    val warmupException = exception?.message
+                    logger.e("Error warming up integrity token provider | warmupException = $warmupException", RadarLogType.SDK_ERROR)
```

Comment:
@tderouin We should pass the exception as the third param to `e`. This is a good use of `SDK_ERROR`